app/src/main/java/com/github/swent/echo/compose/association/AssociationMainScreen.kt

```diff
@@ -18,15 +37,78 @@ fun AssociationMainScreen(
     addAssociationToFilter: (Association) -> Unit,
     followedAssociations: List<Association>,
     committeeAssociations: List<Association>,
+    eventsFilter: List<Association>,
+    isOnline: Boolean
+) {
+    Column(modifier = Modifier.fillMaxSize().testTag("association_main_screen")) {
```

Comment:
This column should be scrollable so that we can see the list of event even if the associations list is expanded.

Reply:
I can change that, thanks